docs/.vuepress/public/favicon-lm-dm.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="144" height="144" viewBox="0 0 144 144" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <style>
+    path {
+      fill: #1C1C1A;
+    }
+    @media (prefers-color-scheme: dark) {
+      path {
+        fill: white;
+      }
+    }
```

Comment:
This style tag provides some basic lightmode/darkmode detection. The media query listens for your OS's preference and changes the fill accordingly.